bitnami/odoo/templates/svc.yaml

```diff
@@ -41,6 +41,15 @@ spec:
       {{- else if eq .Values.service.type "ClusterIP" }}
       nodePort: null
       {{- end }}
+    - name: longpolling
+      port: {{ coalesce .Values.service.ports.longpolling .Values.service.port }}
+      protocol: TCP
+      targetPort: longpolling
+      {{- if and (or (eq .Values.service.type "NodePort") (eq .Values.service.type "LoadBalancer")) (not (empty (coalesce .Values.service.nodePorts.longpolling .Values.service.nodePort))) }}
+      nodePort: {{ coalesce .Values.service.nodePorts.longpolling .Values.service.nodePort }}
+      {{- else if eq .Values.service.type "ClusterIP" }}
+      nodePort: null
+      {{- end }}
```

Comment:
Same comment here regarding `coalesce`. I would only add the service if there's a service and container port for longpolling.